src/backup/background/procedures/restore/index.test.ts

```diff
@@ -183,4 +183,37 @@ describe('BackupRestoreProcedure', () => {
 
         expect(boom).rejects.toThrow('Muahaha!')
     })
+
+    it('should not restore empty objects in place of Blobs', async () => {
+        const favChange = {
+            operation: 'create',
+            object: { favIcon: {}, hostname: 'test.com' },
+            collection: 'favIcons',
+        }
+        const pageChange = {
+            operation: 'create',
+            object: {
+                url: 'test.com/route',
+                screenshot: {},
+                hostname: 'test.com',
+            },
+            collection: 'pages',
+        }
+
+        const createObject = jest.fn()
```

Comment:
Please don't use any Jest stuff in new code, but prefer Sinon or just manual mocks instead.